src/routes/__tests__/express.test.js

```diff
@@ -30,6 +28,41 @@ describe('Basic API', () => {
         });
     });
 
+    it('returns sharing disabled for config if no secrets specified', async () => {
```

Comment:
Add a test that shows `body` with actual config?

Reply:
the next test has it? it's a pretty empty config but its there